### PR TITLE
Milestone 4 Portfolio

### PR DESCRIPTION
**Pull request by RodChak**
# [Portfolio milestone  4](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md): Desktop version

### In this milestone, I did the following:
- Used media queries to make sure that my website displays correctly on desktop screens.
- Updated README.md file
- Added another stylesheet file for the desktop version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,152 +8,219 @@
...
-                <div class="social-media-icons">
-                    <ul class="social-media-icons">
+                <div class="split">
+                    <h2>About <br> Myself</h2>
+                    <p>Hello I’m a software developer! I can help you build a product, feature or website Look through
+                        some
+                        of my work and experience! If you like what you see and have a project you need coded, don’t
+                        hestiate to contact me.</p>
+                    <h6><a href="#">LET’S CONNECT</a></h6>
+                    <ul class="social-media-icons_2">
```
Comment (Kandy-Peter) [inline index.html:211]: - [x] I would suggest you increase the `padding-top` for your `about` section and also remove this empty blank bottom space to stick the Figma design.
**1. Your design**
Screenshot from 2022-07-09 14-24-25

**2. Figma design:**
Screenshot from 2022-07-09 14-33-41

File: index.html [hunk 2]
```diff
@@ -8,152 +8,219 @@
...
+                            <li><a href="#"><img src="./images/angelist.svg" alt="angelist"></a></li>
+                        </ul>
+                    </div>
                 </div>
+
             </div>
         </section>
         <section class="Works_Section">
             <div class="card card_1">
-                <img src="./images/Snapshot1.png" alt="Snapshoot_Portfolio_1">
```
Comment (Kandy-Peter) [inline index.html:42]: - [x] Also, ensure that you add more padding for your work section to match the Figma design. I know you can do that :blush:

**1. Your card section:**
Screenshot from 2022-07-09 14-24-03

**2. Figma design**
Screenshot from 2022-07-09 14-40-34

**Review by Kandy-Peter — CHANGES_REQUESTED** [review]
Hi @RodChak,

Good job so far! I know you have worked hard on implementing the changes requested by the previous reviewer. However, there is an issue that you still need to work on to go to the next project but you are almost there!

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
## Required Changes ♻️
- [ ] Kindly check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: desktop_style.css [hunk 3]
```diff
@@ -0,0 +1,458 @@
...
+    column-gap: 1.527vw;
+    height: 496px;
+    margin-bottom: 142px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 24px;
+    padding-top: 114px;
+    padding-left: 1.5vw;
+    padding-right: 1.5vw;
+  }
```
Comment (sumairq) [inline desktop_style.css]: - [ ] The screenshots are not contained within the cards in the works section. Kindly fix this issue and your project is good to go.  Cheers 👍 

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
image  |  image

**Review by sumairq — CHANGES_REQUESTED** [review]
Hi @RodChak , 



Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: desktop_style.css (hunk 3 above)
Comment (RodChak) [reply desktop_style.css:147]: Hey Sumairq, thank you for the code review. I just wanted to clarify something real quick before I proceed, is your review comment more like a suggestion that I should contain the screenshots within the cards in the works section, or do you mean the screenshots should not be in the container within the cards in the works section. I'm a little bit confused, please may you clarify this for me. I have attached a screenshot of the code.
 image
As of now, the screenshot images in my code are contained within the "card" container in the works section, so I'm assuming you mean I should remove them out of the "card" container? Is that right?

**Review by ishimwezachee — APPROVED** [review]
Hi @RodChak , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
PR is well described 👍 
The project is running successfully 👍 
Implimented the changes requested by previous reviewers 👍 
linters are passing 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**